docs/wiki/Cryptography/keccak256.md

```diff
@@ -0,0 +1,23 @@
+# Keccak256
+
+Keccak256 is a cryptographic hash function prominently used in the Ethereum blockchain. 
+
+## Brief History
+
+Keccak256 originated as a submission to the [NIST cryptographic hash algorithm competition](https://csrc.nist.gov/projects/hash-functions/nist-hash-function-competition). This competition aimed to identify a new secure hash algorithm to replace SHA-1. The KECCAK algorithm was designed by a team comprising Guido Bertoni, Joan Daemen, Michaël Peeters, and Gilles Van Assche. Their design was notable for its ability to absorb a phrase of arbitrary length and produce a fixed-size output. 
```

Comment:
> Their design was notable for its ability to absorb a phrase of arbitrary length and produce a fixed-size output.

This is property of most hash function and not really specific to keccak. Maybe clarify what's the difference between their original winning submission used in eth and final sha3 version.

Reply:
You can also elaborate or add a section on keccak design, mention it's using a sponge function

Reply:
Maybe also point out that even though the opcode is called `SHA3`, it computes Keccak-256 and the NOT the finalized standard (FIP-202 SHA-3). Related discussion: https://github.com/ethereum/EIPs/issues/59.

An empty vector is used to differentiate between them:
```
Keccak256("") =
  c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470
```
```
SHA3("") =
  a7ffc6f8bf1ed76651c14756a061d662f580ff4de43b49fa82d80a4b80f8434a
```